output/Read Me.txt

```diff
@@ -1,5 +1,7 @@
 Open *demo.html* to see a list of all the glyphs in your font along with their codes/ligatures.
 
+To use the generated font in desktop programs, you can install the TTF font. In order to copy the character associated with each icon, refer to the text box at the bottom right corner of each glyph in demo.html. The character inside this text box may be invisible; but it can still be copied. See this guide for more info: https://icomoon.io/#docs/local-fonts
```

Comment:
FWIW in actual use you'll never really use the characters but refer to the classes like `.icon-linkedin`